hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMDirectoriesPurgeRequestAndResponse.java

```diff
@@ -195,9 +195,19 @@ private OMRequest createPurgeKeysRequest(String fromSnapshot, String purgeDelete
   private PurgePathRequest wrapPurgeRequest(
       final long volumeId, final long bucketId, final String purgeDeletedDir,
       final List<OmKeyInfo> purgeDeletedFiles, final List<OmKeyInfo> markDirsAsDeleted) {
+    return wrapPurgeRequest(
```

Comment:
why is this needed?

Reply:
Two helper functions don't require ranges, and my added tests require a range parameter, which is why it's needed.